tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +50,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="num_workers", default=NUM_WORKERS, type="int",
+                      help="override number of worker threads", metavar="NUM_WORKERS")
+    parser.add_option("-t", "--timeout",
```

Comment:
It previously was, and I used `-1` to mean "default". You asked me to use `None` for default instead (I agree), but that doesn't allow using `Float` as a type anymore, because it will fail converting `None` to `Float`.